src/io.ts

```diff
@@ -82,12 +83,16 @@ export function outputResult(
       abiOutPath,
       JSON.stringify(ast.solidityABI.contracts[solFilePath][contractName]['abi'], null, 2),
     );
-    outputFileSync(fullCodeOutPath, code);
-
+    outputFileSync(
+      _transform ? replaceSuffix(fullCodeOutPath, '_warp.cairo') : fullCodeOutPath,
```

Comment:
Can we just push fullCodeOutPath generation outside the function? So we won't need the condition and an extra argument

Reply:
No, we can't get true `contractName` at line 79, if we move the logic outside the function and without an extra argument.

Reply:
But contractName is not used to construct fullCodeOutPath. Only OutputPath and options.outputDir are

Reply:
Yeah, but still we have to add an extra argument which enables different file output paths between `transpile` and `transform` commands.

Reply:
I've pushed changes. It works on my side, please verify

Reply:
did you provide the optional output dir argument in `bin/warp transform` command?

Reply:
Because I'm getting the following error
<img width="1022" alt="Screenshot 2023-02-08 at 4 41 21 PM" src="https://user-images.githubusercontent.com/47271616/217513805-dccd5ebf-3f46-49db-855f-9f9edcbf391d.png">

Reply:
I didn't test it for the `-o` option. Now it should be fixed